modules/core/www/frontpage_federation.php

```diff
@@ -8,6 +8,8 @@
 if ($config->getBoolean('admin.protectindexpage', false)) {
     \SimpleSAML\Utils\Auth::requireAdmin();
 }
+$as = new \SimpleSAML\Auth\Simple('admin');
+$logouturl = $as->getLogoutURL();
```

Comment:
Is it logical to have a `getAdminLoginURL()` but lot a getAdminLogoutURL? Seems you’d either want both functions or none.

Reply:
Done